Simbody/include/simbody/internal/SmoothSphereHalfplaneContact.h

```diff
@@ -82,38 +82,38 @@ class SmoothSphereHalfplaneContactImpl;
  * value of the slope at low velocities.
  *
  * The original Hertz force between a sphere and a plane is given by:
- *      fh = 4/3*k*x*(R*k*x)^(1/2)
- * where k = 0.5*stiffness^(2/3) where stiffness is the effective Young's
+ * <pre>     fh = (4/3) k x (R k x)^(1/2) </pre>
+ * where k = 0.5 stiffness^(2/3) where stiffness is the effective Young's
  * modulus, which is assumed identical for both contacting materials (i.e.,
  * sphere and plane), x is penetration depth, and R is sphere radius.
  * In the smooth approximation, we use the expression:
- *      fh_pos = 4/3*k*(R*k*)^(1/2)*((x^2+eps)^(1/2))^(3/2)
- *      fh_smooth = fh_pos*(1./2.+(1./2.)*tanh(bd*x));
+ * <pre>     fh_pos = (4/3) k (R k)^(1/2) ((x^2+eps)^(1/2))^(3/2) </pre>
+ * <pre>     fh_smooth = fh_pos (1/2+(1/2)tanh(b d x)); </pre>
```

Comment:
Could you remove the semicolon in the end?Note that we might have to remove that part anyways based on Sherm's comment.